nncf/common/quantization/initialization/range.py

```diff
@@ -160,6 +160,28 @@ def __init__(self, is_weights: bool, mode: QuantizationMode, per_channel: bool):
         self._mode = mode
         self._per_channel = per_channel
 
+    @property
+    def is_weights(self) -> bool:
+        """
+        Returns boolean that defines tensor type.
+        True for Weights, False for Activations.
+        """
+        return self._is_weights
+
+    @property
+    def mode(self) -> QuantizationMode:
+        """
+        Returns quantization mode: symmetric or asymmetric.
+        """
+        return self._mode
```

Comment:
Why are we still trying to implement `mode` instead of the `scheme`? `QuantizationMode` is not `symmetric or asymmetric` anymore. I hope that @alexsu52 can clear it here.

Reply:
Good catch, this name is from legacy quantization class, we can rename it

Reply:
When and where the renaming would be done?

Reply:
Let's discuss it offline with the team

Reply:
Done, please check